zomes/blog/capabilities/main/manifest.json

```diff
@@ -0,0 +1,42 @@
+{
+  "capability": {
+    "membrane": "agent"
+  },
+
+  "fn_declarations": [
+    {
+        "name": "create_post",
+        "signature": {
+          "inputs": [
+            {"name": "content", "type": "string"},
+            {"name": "in_reply_to", "type": "hash"}
+          ],
+          "outputs": [
+            {"name": "hash", "type": "string"}
+          ]
+        }
```

Comment:
And as @sphinxc0re mentioned earlier, we should get rid of ```signature``` and have ```name```, ```inputs``` and ```outputs``` at the same level.
(I didn't want to introduce too many changes at once)